NetKAN/MusicMuteContinued.netkan

```diff
@@ -1,12 +1,18 @@
 {
     "spec_version" : 1,
```

Comment:
The `1` needs to be changed to `"v1.4"` in order to use `find` in the install stanza.